modules/cluster-and-ha-management/pages/ha-overview.adoc

```diff
@@ -0,0 +1,70 @@
+:toc:
+= High Availability Overview
+:description: Overview of High Availability functionality and supported features.
+
+A TigerGraph system with High Availability (HA) is a cluster of server machines which uses replication.
+Allowing for the database to continue to run with minimal impact to operations until the offline nodes are restored or removed permanently.
```

Comment:
```suggestion
A TigerGraph system with High Availability (HA) is a cluster of server nodes which uses replication to provide continuous operation of most services in the event that one or more nodes is offline. 
```
Your split into two sentences is not grammatical. 
I think "continuous operation of most services" or "many services" is a good way to say what we do.

Reply:
thank you fixed this with latest commit.